internal/broker/helper_test.go

```diff
@@ -49,6 +54,23 @@ func newBrokerForTests(t *testing.T, cfg *brokerForTestConfig) (b *broker.Broker
 	if cfg.allowedSSHSuffixes != nil {
 		cfg.SetAllowedSSHSuffixes(cfg.allowedSSHSuffixes)
 	}
+	if cfg.allowedUsers == nil {
+		cfg.SetAllowedUsers(map[string]struct{}{"OWNER": {}})
+	} else {
+		cfg.SetAllowedUsers(cfg.allowedUsers)
+	}
```

Comment:
We don't add "OWNER" to the `userConfig.allowedUsers` map anymore

```suggestion
	if cfg.allowedUsers != nil {
		cfg.SetAllowedUsers(cfg.allowedUsers)
	}
```